jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglCanvas.java

```diff
@@ -497,7 +497,6 @@ protected void createContext(AppSettings settings)  {
         // This will activate the "effective data" scrubber.
         showGLDataEffective.set(settings.getBoolean("GLDataEffectiveDebug"));
         
-        glData.depthSize = settings.getBitsPerPixel();
```

Comment:
Hi @JNightRide ,
How does this line removal relates to solving the gradle zip build issue?